test/integration/tarantool_role_test.lua

```diff
@@ -0,0 +1,155 @@
+local t = require('luatest')
+local fio = require('fio')
+
+local helpers = require('test.helper')
+local Server = require('test.helper_server')
+
+local g = t.group('tarantool_role_integration_test')
+local tmpdir = fio.tempdir()
```

Comment:
It's better to create a directory in `before_all` hook and not on require. Otherwise cleanup is not guaranteed.

Reply:
changed